library/Rules/Not.php

```diff
@@ -23,6 +23,13 @@ public function __construct(Validatable $rule)
         $this->rule = $rule;
     }
 
+    public function setName($name)
+    {
+        $this->rule->setName($name);
+
+        parent::setName($name);
```

Comment:
`setName()` must return self as the same of `AbstractRule`, please, `return $this` in the end of the method.